crates/linker/src/pe.rs

```diff
@@ -614,11 +632,11 @@ impl Preprocessor {
                 characteristics: Default::default(),
             };
 
-            let header_array: [u8; std::mem::size_of::<ImageSectionHeader>()] =
-                unsafe { std::mem::transmute(header) };
+            let header_array: [u8; W] = unsafe { std::mem::transmute(header) };
+
+            result[self.extra_sections_start + i * W..][..W].copy_from_slice(&header_array);
```

Comment:
Might be nice to create a variable for `self.extra_sections_start + i * W`. I always find complex expressions in front of `..` tricky to read. Not important though!

Reply:
good idea, I'll fold that into another branch